### ISSUE TITLE
Add Link Check Results to CLI Output

### ISSUE THREAD

**Issue by marshalmiller** [marshalmiller/linkrot#120]
Right now, if you use the -o argument to export the results to a text file, the document metadata and the list of links are the only components listed. I would like to add the results of the link check to this output as well.

**Comment by mailtodanish** [comment]
I will work on this.

**Comment by mailtodanish** [comment]
`$ linkrot  -c -o test.txt 3.pdf`
@marshalmiller  expected txt file would be as below?
```
Document info:
- Author = abecker
- CreationDate = D:20141124102132-05'00'
- Creator = PScript5.dll Version 5.2.2
- ModDate = D:20141124102153-05'00'
- Pages = 24
- Producer = Acrobat Distiller 9.5.5 (Windows)
- Title = ET_17-2.indd
- dc = {'format': 'application/pdf', 'title': {'x-default': 'ET_17-2.indd'}, 'creator': ['abecker']}
- pdf = {'Producer': 'Acrobat Distiller 9.5.5 (Windows)'}
- rdf = {'li': 'abecker'}
- xap = {'ModifyDate': '2014-11-24T10:21:53-05:00', 'CreateDate': '2014-11-24T10:21:32-05:00', 'MetadataDate': '2014-11-24T10:21:53-05:00', 'CreatorTool': 'PScript5.dll Version 5.2.2'}
- xapmm = {'DocumentID': 'uuid:6685251d-ac25-4eb2-8093-7183ea746a46', 'InstanceID': 'uuid:5ac526be-f93d-41f5-8260-f46c3dda0f96'}

References: 6
- URL: 6

Tip: You can use the '-v' flag to see all references
Summary of link checker:
[92m3 working[0m
[91m1 broken (reason: 404)[0m
  - http://dx.doi.org/10.1353/ (page 24)
[91m2 broken (reason: [Errno 11001] getaddrinfo failed)[0m
  - http://justus.anglican (page 24)
  - http://dx.doi (page 24)
```

**Comment by marshalmiller** [comment]
@mailtodanish Yes, exactly.

**Comment by marshalmiller** [comment]
Addressed in PR #145.
